src/cpp/shared/combat/army.cpp

```diff
@@ -1648,6 +1976,9 @@ void army::MoveTo(int hexIdx) {
 }
 
 void army::MoveAttack(int targHex, int x) {
+  gMoveAttack = false;
+  if(x == 1 || (!x && gpCombatManager->combatGrid[targHex].unitOwner != -1))
```

Comment:
If you're using x, then give it a more descriptive variable name.

Reply:
It's something like a flag that tells that the moveattack is just move or move-and-attack. But it works differently for AI turns that's why I didn't rename it

Reply:
Oy. Then comment and explain what it does.